components/mpas-ocean/src/shared/mpas_ocn_surface_bulk_forcing.F

```diff
@@ -592,8 +597,82 @@ subroutine ocn_surface_bulk_forcing_active_tracers(meshPool, forcingPool, tracer
 
    end subroutine ocn_surface_bulk_forcing_active_tracers!}}}
 
-end module ocn_surface_bulk_forcing
+!***********************************************************************
+!
+!  routine ocn_surface_bulk_forcing_debug_tracers
+!
+!> \brief   Determines the debug tracers forcing array used for the bulk forcing.
+!> \author  Kat Smith
+!> \date    01/25/2022
+!> \details
+!>  This routine computes the debug tracers forcing arrays used later in MPAS.
+!
+!-----------------------------------------------------------------------
+
+   subroutine ocn_surface_bulk_forcing_debug_tracers(meshPool, forcingPool, tracerGroup,  &
+      tracersSurfaceFlux, err)
+
+      !-----------------------------------------------------------------
+      !
+      ! input variables
+      !
+      !-----------------------------------------------------------------
+      type (mpas_pool_type), intent(in) :: meshPool !< Input: mesh information
+
+      !-----------------------------------------------------------------
+      !
+      ! input/output variables
+      !
+      !-----------------------------------------------------------------
+      type (mpas_pool_type), intent(inout) :: forcingPool !< Input: Forcing information
+      real (kind=RKIND), dimension(:,:), intent(inout) :: tracersSurfaceFlux
+      real (kind=RKIND), dimension(:,:,:), intent(inout) :: tracerGroup
+
+      !-----------------------------------------------------------------
+      !
+      ! output variables
+      !
+      !-----------------------------------------------------------------
 
+      integer, intent(out) :: err !< Output: Error flag
+
+      !-----------------------------------------------------------------
+      !
+      ! local variables
+      !
+      !-----------------------------------------------------------------
+
+      integer :: iCell, nCells
+      integer, pointer :: index_debug_flux
+      integer, dimension(:), pointer :: nCellsArray
+
+      type(mpas_pool_type),pointer :: tracersSurfaceFluxPool
+
+      real (kind=RKIND), dimension(:), pointer :: seaIceFreshWaterFlux
+
+      err = 0
+
+      call mpas_pool_get_dimension(meshPool, 'nCellsArray', nCellsArray)
+      call mpas_pool_get_subpool(forcingPool, 'tracersSurfaceFlux',tracersSurfaceFluxPool)
+      
+      call mpas_pool_get_dimension(tracersSurfaceFluxPool, 'index_tracer1SurfaceFlux', index_debug_flux)
+      
+      call mpas_pool_get_array(forcingPool, 'seaIceFreshWaterFlux', seaIceFreshWaterFlux)
+
+      nCells = nCellsArray( 3 )
+
+      ! Build surface fluxes at cell centers
+      !$omp parallel
+      !$omp do schedule(runtime) 
+      do iCell = 1, nCells
+        tracersSurfaceFlux(index_debug_flux, iCell) = seaIceFreshWaterFlux(iCell)
```

Comment:
@katsmith133  note here that `seaIceFreshWaterFlux` has units of kg m^(-2) s^(-1), I would suggest making the source normalized by rho_sw*layerthickness(1,iCell)

Reply:
Done! Here is what it looks like now:
![DebugTracerSeaIceFluxTest](https://user-images.githubusercontent.com/5370137/151643300-788ed30a-ea53-4abb-8a6d-d63d70fbc10a.png)